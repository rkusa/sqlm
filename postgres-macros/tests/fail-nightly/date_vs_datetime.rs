../fail-stable/date_vs_datetime.rs