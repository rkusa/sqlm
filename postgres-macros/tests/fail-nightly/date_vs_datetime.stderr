error[E0271]: type mismatch resolving `<OffsetDateTime as SqlType>::Type == Date`
 --> tests/fail-nightly/date_vs_datetime.rs:5:62
  |
5 |     let _: time::OffsetDateTime = sql!("SELECT NOW()::DATE").await.unwrap();
  |                                                              ^^^^^ expected `Date`, found `OffsetDateTime`
  |
  = note: required for `Sql<'_, Primitive<time::Date>, OffsetDateTime>` to implement `IntoFuture`
